README.md

```diff
@@ -1,7 +1,7 @@
 # Torch.jl
 Sensible extensions for exposing torch in Julia.
 
-This package is aimed at providing the `Tensor` type, which offloads all computations over to [PyTorch](https://pytorch.org).
+This package is aimed at providing the `Tensor` type, which offloads all computations over to Torch.
```

Comment:
Best to keep the pytorch name there and disclaim that it isn't using python, since we do want to credit it appropriately.